src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -18,6 +18,7 @@
     public final static String JSON_DIRECTORY = "json/";
 
     protected static final String SAMPLE_JSON = "sample.json";
+    protected static final String SCENARIO_OUTLINE = "scenario-outline.json";
```

Comment:
I don't want to new test file, problematic cases should be merged with sample.json